src/ServiceControl.Audit/Infrastructure/CheckRavenDBIndexLag.cs

```diff
@@ -0,0 +1,90 @@
+﻿namespace ServiceControl
+{
+    using System;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using NServiceBus.CustomChecks;
+    using NServiceBus.Logging;
+    using Raven.Client;
+    using Raven.Abstractions.Data;
+    using ServiceControl.Audit.Infrastructure.Settings;
+
+    class CheckRavenDBIndexLag : CustomCheck
+    {
+        public CheckRavenDBIndexLag(IDocumentStore store, LoggingSettings settings)
+            : base(nameof(CheckRavenDBIndexLag), "Storage", TimeSpan.FromMinutes(5))
```

Comment:
You need a unique name otherwise you can't really distinguish those on the UI. I would also prefer the name hard-coded as a string so that when the name of the class changes the public visible name that is exposed on the SP page doesn't